modules/cluster/templates/startup.sh.tpl

```diff
@@ -17,24 +17,25 @@ if [ ! -z '${custom_http_proxy}' ]; then
   export https_proxy=$http_proxy
 fi
 
+# Get Vault up and running as quickly as possible to get the auto-heal health
+# check passing.  This results in faster recovery and faster rolling upgrades.
+
 # Deps
 export DEBIAN_FRONTEND=noninteractive
-apt-get update -yqq
-apt-get upgrade -yqq
-apt-get install -yqq jq libcap2-bin logrotate unzip
-
-# Install Stackdriver for logging and monitoring
-curl -sSfL https://dl.google.com/cloudagents/install-logging-agent.sh | bash
-curl -sSfL https://dl.google.com/cloudagents/install-monitoring-agent.sh | bash
+# apt-get upgrade -yqq
+# apt-get install -yqq jq libcap2-bin logrotate unzip
```

Comment:
Is `logrotate` no longer needed? Seems we're still using it below via the `logrotate.d` directory.

Also if a line in the script shouldn't be executed, please just remove as opposed to commenting it out.

Reply:
Good catch, I'll add that back in.